src/actions/ServerApiActions.js

```diff
@@ -0,0 +1,26 @@
+import {
+  SET_IS_FETCHING_ERROR,
+  SET_IS_FETCHING_IN_PROGRESS,
+  SET_IS_TOKEN_RECEIVED,
+  SET_TOKEN_EXPIRATION_TIME
+} from '../actionTypes';
+
+export const setIsTokenReceived = isTokenReceived => ({
+  type: SET_IS_TOKEN_RECEIVED,
+  payload: isTokenReceived
```

Comment:
If you go with payload, then I'd suggest to make it an object, so that for other cases you would have an ability to pass more data